### PR TITLE
Pa home UI formatting

### PR DESCRIPTION
**Pull request by borisbauer-unity**
### Description

Simple PA home UI layout changes, following the recent UI design proposal.

- some small changes to put some words in bold
- added large Welcome title and smaller "Configurations" title above interactive UI elements (drop-downs, buttons)
- added a separator between the welcome text area and interactive UI elements

### CODE REVIEW

File: Editor/UI/Framework/SharedStyles.cs [hunk 1]
```diff
@@ -128,6 +131,35 @@ public static GUIStyle TextAreaWithDynamicSize
             }
         }
 
+        public static GUIStyle WelcomeLabel
```
Comment (mtrive) [inline Editor/UI/Framework/SharedStyles.cs:134]: "WelcomeLabel" sounds specific to the `ProjectAuditorWindow`. Can we generalize it by naming it "TitleLabel" ?

File: Editor/UI/Framework/SharedStyles.cs [hunk 2]
```diff
@@ -128,6 +131,35 @@ public static GUIStyle TextAreaWithDynamicSize
...
+        {
+            get
+            {
+                if (s_WelcomeLabel == null)
```
Comment (mtrive) [inline Editor/UI/Framework/SharedStyles.cs:138]: same for `s_WelcomeLabel `

**Review by mtrive — APPROVED** [review]
Just a minor comment.

Thanks for this.

File: Editor/UI/Framework/SharedStyles.cs (hunk 1 above)
Comment (borisbauer-unity) [reply Editor/UI/Framework/SharedStyles.cs:134]: > "WelcomeLabel" sounds specific to the `ProjectAuditorWindow`. Can we generalize it by naming it "TitleLabel" ?

Yes, makes sense.

File: Editor/UI/Framework/SharedStyles.cs (hunk 2 above)
Comment (borisbauer-unity) [reply Editor/UI/Framework/SharedStyles.cs:138]: Yes, as the previous one, let's rename it.

**Review by mtrive — APPROVED** [review]
Thanks!

**Pull request merged**